app/services/update_course_stats.rb

```diff
@@ -6,9 +6,9 @@
 require_dependency "#{Rails.root}/lib/data_cycle/update_logger"
 require_dependency "#{Rails.root}/lib/analytics/histogram_plotter"
 require_dependency "#{Rails.root}/lib/importers/revision_score_importer"
-require_dependency "#{Rails.root}/lib/importers/average_views_importer"
-require_dependency "#{Rails.root}/lib/errors/update_service_error_helper"
-require_dependency "#{Rails.root}/lib/data_cycle/course_queue_sorting"
+require_dependency Rails.root.join('lib', 'importers', 'average_views_importer')
```

Comment:
This seems unrelated.

Reply:
Let me fix it, thanks.

Reply:
@ragesoss I have fixed it, and now only 2 files have the necessary changes.